source/glossary.rst

```diff
@@ -16,7 +16,7 @@ it without the accompanying context provided by the rest of this documentation.
      that directory.
 
    Artifact
-     A QIIME 2 :term:`Result` that countains data to operate on.
```

Comment:
![image](https://user-images.githubusercontent.com/274668/126415875-48928383-8523-40bc-bd22-95330aa0d54f.png)

Reply:
[I count the spiders on the wall.
I count the cobwebs in the hall.
I count the candles on the shelf.
When I'm alone, I count myself.](https://www.youtube.com/watch?v=ZIniljT5lJI)